saleor/graphql/account/types.py

```diff
@@ -82,19 +82,35 @@ class AddressInput(BaseInputObjectType):
 
 @federated_entity("id")
 class Address(ModelObjectType[models.Address]):
-    id = graphene.GlobalID(required=True)
-    first_name = graphene.String(required=True)
-    last_name = graphene.String(required=True)
-    company_name = graphene.String(required=True)
-    street_address_1 = graphene.String(required=True)
-    street_address_2 = graphene.String(required=True)
-    city = graphene.String(required=True)
-    city_area = graphene.String(required=True)
-    postal_code = graphene.String(required=True)
+    id = graphene.GlobalID(required=True, description="The ID of the address.")
+    first_name = graphene.String(
+        required=True, description="The given name of the address."
+    )
+    last_name = graphene.String(
+        required=True, description="The family name of the address."
+    )
+    company_name = graphene.String(
+        required=True, description="Company or organization name."
+    )
+    street_address_1 = graphene.String(
+        required=True, description="The first line of the address."
+    )
+    street_address_2 = graphene.String(
+        required=True, description="The second line of the address."
+    )
+    city = graphene.String(required=True, description="The city of the address.")
+    city_area = graphene.String(
+        required=True, description="The district of the address."
+    )
+    postal_code = graphene.String(
+        required=True, description="The postal code of the address."
+    )
     country = graphene.Field(
-        CountryDisplay, required=True, description="Shop's default country."
+        CountryDisplay, required=True, description="The country of the address."
+    )
+    country_area = graphene.String(
+        required=True, description="The country area of the address."
     )
-    country_area = graphene.String(required=True)
     phone = graphene.String()
```

Comment:
Missing `phone` description; we could add it and mention that phone numbers are validated with Google's [libphonenumber](https://github.com/google/libphonenumber) library and provide a link to it. We use a package this is a Python port of the Google's one. People often ask about Saleor phone validation rules so I guess we should look through all `phone` fields and add a similar description.